migrations/20220907164245_imported_at_publics.up.sql

```diff
@@ -46,4 +46,48 @@ ADD COLUMN _mergestat_synced_at timestamp with time zone  DEFAULT now() NOT NULL
 ALTER TABLE public.trivy_repo_scans
 ADD COLUMN _mergestat_synced_at timestamp with time zone DEFAULT now() NOT NULL;
 
+CREATE  OR REPLACE VIEW public.git_branches AS
+ SELECT git_refs.repo_id,
+    git_refs.full_name,
+    git_refs.hash,
+    git_refs.name,
+    git_refs.remote,
+    git_refs.target,
+    git_refs.type,
+    git_refs.tag_commit_hash,
+    git_refs._mergestat_synced_at
+   FROM public.git_refs
+  WHERE (git_refs.type = 'branch'::text);
+
+
+CREATE OR REPLACE VIEW public.git_tags AS
+ SELECT git_refs.repo_id,
+    git_refs.full_name,
+    git_refs.hash,
+    git_refs.name,
+    git_refs.remote,
+    git_refs.target,
+    git_refs.type,
+    git_refs.tag_commit_hash,
+    git_refs._mergestat_synced_at
+   FROM public.git_refs
+  WHERE (git_refs.type = 'tag'::text);
+
+CREATE OR REPLACE VIEW trivy_repo_vulnerabilities AS
+SELECT
+	trivy_repo_scans.repo_id,
+	r.value->>'Target' as target,
+	r.value->>'Class' as class,
+	r.value->>'Type' as type,
+	v as vulnerability,
+	v->>'VulnerabilityID' as vulnerability_id,
+	v->>'PkgName' as vulnerability_pkg_name,
+	v->>'InstalledVersion' as vulnerability_installed_version,
+	v->>'Severity' as vulnerability_severity,
+	v->>'Title' as vulnerability_title,
+	v->>'Description' as vulnerability_description,
+    trivy_repo_scans._mergestat_synced_at
```

Comment:
```suggestion
        trivy_repo_scans._mergestat_synced_at
```

Reply:
now this is interesting , in my local i have it formatted  .
![Uploading Screen Shot 2022-09-07 at 12.01.54 PM.png…]()